lib/webmachine/test/session.rb

```diff
@@ -56,20 +64,51 @@ def do_request(method, uri, options)
         add_query_params(uri, options[:params])
 
         # Set some default headers and merge the provided ones.
-        @headers['Host'] = [uri.host, uri.port].compact.join(':') unless @headers['Host']
-        @headers['Accept'] = '*/*' unless @headers['Accept']
```

Comment:
@rfestag Ah, damn. I missed the `||=` change. Sorry for the noise...

It's fine. :smiley: